src/main/java/Duke.java

```diff
@@ -1,10 +1,282 @@
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Event;
+import duke.task.Deadline;
+import duke.exceptions.DukeException;
+import duke.exceptions.EmptyInputException;
+import duke.exceptions.IllegalInputException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    public static final String LINE = "____________________________________________________________\n";
```

Comment:
Good use of constants! You can consider using more constants for the rest of your code too!